guides/tutorials/codelabs/azure-kubernetes-source-to-prod/index.md

```diff
@@ -0,0 +1,44 @@
+
+## Setup
+
+Fork [this repository](https://github.com/azure-devops/spin-kub-demo) with a simple 'Hello World' application written in Go. You will commit a change to this fork in order to trigger the Source to Prod pipeline.
+
+## Deploy Dev Ops VM
+
+Run the following command to deploy the 'Continuous Delivery to Kubernetes' [Quickstart Template](https://github.com/Azure/azure-quickstart-templates/tree/master/301-jenkins-acr-spinnaker-k8s) in Azure. It will make sure you have the [Azure CLI 2.0](https://docs.microsoft.com/cli/azure/install-azure-cli) installed and interactively ask if you want to change any of the default behavior. **Important:** When asked, make sure to specify your fork of the sample git repository so that you can trigger the pipeline with your own commits.
+
+```bash
+bash <(curl -sL https://aka.ms/DeployDevOps)
+```
+
+The script should list several outputs after it has finished. Run the 'SSH' output to connect to your Jenkins and Spinnaker instances hosted on the same VM.
+
+## Verify Sample Pipeline
+
+Navigate to [localhost:8080](http://localhost:8080) to see your Jenkins dashboard. Follow the default Jenkins 'Getting Started' instructions and verify the sample 'Basic Docker Build' job successfully built and pushed to your Azure Container Registry as shown in the image below:
+
+![Verify Basic Docker Build](basic-docker-build.png)
+
+Navigate to [localhost:9000](http://localhost:9000) to see your Spinnaker dashboard. Go to 'Applications > spinkubdemo > Pipelines' and verify the pipeline has already started. It should be waiting at the manual judgement stage after already deploying to the staging environment as shown in the image below. Follow the instructions for the manual judgement and click 'Continue' after you have verified the staging environment.
+
+![Verify Kubernetes Pipeline](k8s-pipeline.png)
+
+> NOTE: The 'Destroy Previous Dev Cluster' stage is expected to have failed because there is no previous cluster yet
+
+After the pipeline finishes, go to the Cluster view to inspect your production environment. Unlike the 'dev' cluster, the 'prod' cluster will have a publicly available Ingress IP listed where you can verify your app. Click the icon for the load balancer under the 'prod' cluster as shown in the image below:
+
+![Verify Production IP](prod-ip.png)
+
+## Trigger Sample Pipeline
+
+Finally, make a change to your sample app in git and push your changes (for example, you can change the color or wording in the 'content/index.html' file). Follow the same steps above to watch as your new image is built and deployed to your staging/production environments.
```

Comment:
Maybe add a link to the Jenkins' GitHub plugin for further reading: https://wiki.jenkins-ci.org/display/JENKINS/GitHub+Plugin